.github/workflows/make.yml

```diff
@@ -0,0 +1,34 @@
+name: make
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8, 3.9, '3.10']
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        make install
+    - name: Run make
+      run: |
+        make
+    - name: Upload coverage to Codecov
+      if: ${{ matrix.python-version == 3.9 }}
```

Comment:
Ultranit: Do codecov on not a middle version, I would suggest the lowest version.